source/ore/guidelines.rst

```diff
@@ -160,6 +160,19 @@ system (such as a Sponge plugin that translates chat between languages), connect
 If your plugin sends information (e.g. a plugin list, player data, or map data) to external systems, the information collected
 must be listed on the main page (see above).
 
+    **Metrics (Data Collection)**
+
+    If any collected data (often referred to as "stats" or "metrics" data) about the server, such as version of server or plugins as
+    well as usage information, is to be sent to an external service the plugin must first query the Sponge API `MetricsConfigManager`.
+    Documentation on doing so can be found HERE [HALP, NEED LINK HERE]. This API must be checked before each time data is sent, not
+    only once. Plugins may not modify the values the API returns, but may encourage users to make the decision to enable the
+    collection and sending of this data for their plugin. The use of this API counts as the method by which to check if sending this
```

Comment:
Is there a better way to phrase this than "counts as the method"? It seems a little fuzzy to me.
You can probably use "metrics data" for "this data", as you have defined it above.

Reply:
"The use of this API satisfies the guidelines requirement of checking if the user has allowed sending metrics data."

Maybe?

Reply:
Is it even needed? Above, it states (emphasis mine):

> is to be sent to an external service the plugin **must** first query the Sponge API `MetricsConfigManager`

This would imply that is it mandatory to use this API in order to satisfy the guidelines, so this statement is unnecessary.

Reply:
Fair point.